<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenGLQtAppClass</class>
 <widget class="QMainWindow" name="OpenGLQtAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenGLQtApp</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout"/>
    </item>
    <item>
     <widget class="QSlider" name="xSlider">
      <property name="maximum">
       <number>5760</number>
      </property>
      <property name="singleStep">
       <number>16</number>
      </property>
      <property name="pageStep">
       <number>160</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="ySlider">
      <property name="maximum">
       <number>5760</number>
      </property>
      <property name="singleStep">
       <number>16</number>
      </property>
      <property name="pageStep">
       <number>160</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSlider" name="zSlider">
      <property name="maximum">
       <number>5760</number>
      </property>
      <property name="singleStep">
       <number>16</number>
      </property>
      <property name="pageStep">
       <number>160</number>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="openglqtapp.qrc"/>
 </resources>
 <connections/>
</ui>
